packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -240,56 +266,82 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
+        if (
+          item.itemType !== ContextualMenuItemType.Divider &&
+          item.itemType !== ContextualMenuItemType.Header
+        ) {
           totalItemCount++;
         }
       }
       return (
         <Callout
           {...calloutProps}
-          target={ target }
-          targetPoint={ targetPoint }
-          useTargetPoint={ useTargetPoint }
-          isBeakVisible={ isBeakVisible }
-          beakWidth={ beakWidth }
-          directionalHint={ directionalHint }
-          directionalHintForRTL={ directionalHintForRTL }
-          gapSpace={ gapSpace }
-          coverTarget={ coverTarget }
-          doNotLayer={ doNotLayer }
-          className='ms-ContextualMenu-Callout'
-          setInitialFocus={ shouldFocusOnMount }
-          onDismiss={ this.props.onDismiss }
-          bounds={ bounds }
-          directionalHintFixed={ directionalHintFixed }
+          target={target}
+          targetPoint={targetPoint}
+          useTargetPoint={useTargetPoint}
+          isBeakVisible={isBeakVisible}
+          beakWidth={beakWidth}
+          directionalHint={directionalHint}
+          directionalHintForRTL={directionalHintForRTL}
+          gapSpace={gapSpace}
+          coverTarget={coverTarget}
+          doNotLayer={doNotLayer}
+          className="ms-ContextualMenu-Callout"
+          setInitialFocus={shouldFocusOnMount}
+          onDismiss={this.props.onDismiss}
+          bounds={bounds}
+          directionalHintFixed={directionalHintFixed}
         >
-          <div style={ contextMenuStyle } ref={ (host: HTMLDivElement) => this._host = host } id={ id } className={ this._classNames.container }>
-            { title && <div className={ this._classNames.title } role='heading' aria-level={ 1 }> { title } </div> }
-            { (items && items.length) ? (
+          <div
+            style={contextMenuStyle}
+            ref={(host: HTMLDivElement) => (this._host = host)}
+            id={id}
+            className={this._classNames.container}
+          >
+            {title && (
+              <div
+                className={this._classNames.title}
+                role="heading"
+                aria-level={1}
+              >
+                {" "}
+                {title}{" "}
+              </div>
+            )}
+            {items && items.length ? (
               <FocusZone
-                className={ this._classNames.root }
-                direction={ arrowDirection }
-                isCircularNavigation={ true }
+                className={this._classNames.root}
+                direction={arrowDirection}
+                isCircularNavigation={true}
               >
                 <ul
-                  role='menu'
-                  aria-label={ ariaLabel }
-                  aria-labelledby={ labelElementId }
-                  className={ this._classNames.list }
-                  onKeyDown={ this._onKeyDown }
+                  role="menu"
```

Comment:
Turn the VSCode prettier extension off for the office-fabric-react workspace, it's adding a bunch of formatting here. Try to make this change not have any formatting changes

Reply:
Yes let me do that :) I'm also fixing the merge conflicts and I will update